src/operations/indexes.ts

```diff
@@ -153,24 +153,23 @@ export class IndexesOperation extends AbstractOperation<Document> {
 export class CreateIndexesOperation extends CommandOperation<Document> {
```

Comment:
This is `extends CommandOperation<string[]>` now right? Have the methods been changed to reflect this in `collection.ts` as well?

Reply:
updated!